pkg/kritis/review/review.go

```diff
@@ -103,14 +184,19 @@ func (r Reviewer) ReviewISP(images []string, isps []v1beta1.ImageSecurityPolicy,
 
 	for _, isp := range isps {
 		glog.Infof("Validating against ImageSecurityPolicy %s", isp.Name)
-		// Get all AttestationAuthorities in this policy.
-		auths, err := r.getAttestationAuthoritiesForISP(isp)
+		// Get the attestationauthority in this policy.
```

Comment:
nit: inconsistent capitalization for attestationauthority

Reply:
done